README.md

```diff
@@ -1 +1,51 @@
-Neighborhoods.com Analytics Base Docker Image
+# Neighborhoods.com Analytics Base Docker Image
+
+## How to build the image
+
+Use the bash script provided and provide a version number.
+
+```
+source build_and_push_image.sh 0.0.0
+```
+
+## How to use the image
+
+#### Step 1: log into AWS ECR registry
+
+Run `aws ecr get-login --no-include-email`. Its output will be a command
+starting with `docker login ...` that you should copy and execute to log in to
+the AWS ECR registry for our images.
+
+#### Step 2: pull the image
+
+Run the following (having substrituted the correct version number) to pull the
+specified version of the `analytics-base` image to your local machine:
+
```

Comment:
Might also be good to show the command to get `AWS_ACCOUNT_ID` (`aws sts get-caller-identity --output text --query 'Account'`)

Reply:
Sure, but I'm going to make that a separate ticket, since this one was just to move the repo. Incoming shortly.